<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QMainWindow" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/Image/Gucci_logo.webp</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>630</width>
      <height>550</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="lb_left">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>30</y>
       <width>301</width>
       <height>465</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-top-left-radius:25px;
border-bottom-left-radius:25px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_right">
     <property name="geometry">
      <rect>
       <x>295</x>
       <y>30</y>
       <width>330</width>
       <height>465</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Image/Gucci_register.jpg);
border-top-right-radius:25px;
border-bottom-right-radius:25px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_register" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>281</width>
       <height>465</height>
      </rect>
     </property>
     <widget class="QPushButton" name="btn_sw_login_">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>440</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border:none;
</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_password">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>235</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_hidden_pw">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>265</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="hidden_pwd" resource="Image.qrc">
        <normaloff>:/Image/hidden.png</normaloff>:/Image/hidden.png</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_password">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>255</y>
        <width>230</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(195, 188, 191, 0.3);
border-radius:15px;
padding-left:10px;</string>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_firstName">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>75</y>
        <width>105</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(195, 188, 191, 0.3);
border-radius:15px;
padding-left:10px;</string>
      </property>
      <property name="maxLength">
       <number>49</number>
      </property>
      <property name="placeholderText">
       <string>First Name</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lb_login">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>271</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Gabriola</family>
        <pointsize>31</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgba(0, 0, 0,200);
</string>
      </property>
      <property name="text">
       <string>REGISTER</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lb_have_an_acc">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>440</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Have an account?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_signup">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>380</y>
        <width>130</width>
        <height>55</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#btn_signup{
	background-color:rgba(157,150,153,0.86);
	border-radius:20px;
}
QPushButton#btn_signup:hover{
	background-color:rgba(102, 94, 95, 0.21);
	border: 3px solid rgba(157,150,153,0.95);
}
QPushButton#btn_signup:pressed{
	padding-bottom:5px;
	background-color: rgb(67, 67, 67);
	border: 3px solid rgba(157,150,153,0.95);
	
}
</string>
      </property>
      <property name="text">
       <string>Sign Up</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_lastName">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>75</y>
        <width>106</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <italic>false</italic>
        <bold>false</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(195, 188, 191, 0.3);
border-radius:15px;
padding-left:10px;</string>
      </property>
      <property name="maxLength">
       <number>49</number>
      </property>
      <property name="placeholderText">
       <string>Last Name</string>
      </property>
      <property name="cursorMoveStyle">
       <enum>Qt::LogicalMoveStyle</enum>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_userName">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>135</y>
        <width>230</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(195, 188, 191, 0.3);
border-radius:15px;
padding-left:10px;</string>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lb_first_name">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>55</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>First name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_userName">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>115</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Username:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_last_name">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>55</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Last name:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_confirmPassword">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>320</y>
        <width>230</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(195, 188, 191, 0.3);
border-radius:15px;
padding-left:10px;</string>
      </property>
      <property name="maxLength">
       <number>15</number>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="cursorPosition">
       <number>0</number>
      </property>
      <property name="placeholderText">
       <string>Confirm Password</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lb_confirmPassword">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>300</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm Password:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_hidden_confirm_pw">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>330</y>
        <width>25</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="hidden_pwd" resource="Image.qrc">
        <normaloff>:/Image/hidden.png</normaloff>:/Image/hidden.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_minimize_window">
      <property name="geometry">
       <rect>
        <x>235</x>
        <y>0</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#btn_minimize_window{
	border:none;
}
QPushButton#btn_minimize_window:hover{
	border:none;
	
	
	border-image: url(:/Image/minimize.png);
}
QPushButton#btn_minimize_window:pressed{
	
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_close_window">
      <property name="geometry">
       <rect>
        <x>255</x>
        <y>0</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#btn_close_window{
	border:none;
}
QPushButton#btn_close_window:hover{
	border:none;
	
	border-image: url(:/Image/close_window.png);
}
QPushButton#btn_close_window:pressed{
	
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lb_notice">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>271</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="lb_telephone">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>175</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Telephone:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_telephone">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>195</y>
        <width>230</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(195, 188, 191, 0.3);
border-radius:15px;
padding-left:10px;</string>
      </property>
      <property name="placeholderText">
       <string>Telephone</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections/>
</ui>
